### PR TITLE
Revision of the first few pages

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Anqiqi-Li** [comment]
I try to remove some sentences to simplify the intro and add package references (from xxx import yyy) when we give code examples.
Some general comments:
 - The Overview page under Scenes is still far too long and detailed IMHO. This should be a brief intro to get users to start something practical instead of creating more questions. Perhaps the sections under "Resources" should be in a separate page? 
 - Should change a more appropriate title for the page about the authentication (currently "Kognic APIs/ Kognic APIs"), and move under "Kognic IO".

File: docs-src/docs/kognic-io/pre_annotations.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ see [Creating Multiple Inputs With One Call](working_with_scenes_and_inputs.md#c
 ### 1. Creating a scene
 
 :::note
-The scene is a subset of the input, specifically referring to the data, such as images or point clouds.
-An input is what is created when this data is ready to be annotated.
+The scene is a subset of the input, specifically referring to the raw data, such as images or point clouds.
+An input is created when this data is associated to a request in a project with task definition and guildline, and is ready to be annotated.
```
Comment (olinoskar) [inline docs-src/docs/kognic-io/pre_annotations.md:27]: I think that this note can be removed completely. It's not correct to say that _a scene is a subset of an input_ and it doesn't really add much information other than that

File: docs-src/docs/kognic-io/pre_annotations.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ see [Creating Multiple Inputs With One Call](working_with_scenes_and_inputs.md#c
...
 :::
 
-The interface for creating just a scene, without an input, is the same as we are familiar with.
-The exception is that by not providing a project or a batch in the function call, the scene will be "dangling" until deemed ready for annotation
+The interface for creating a scene is similar to creating an input.
+The exception is to omit a project or a batch in the function call, then the scene will be "dangling" until deemed ready for annotation
```
Comment (olinoskar) [inline docs-src/docs/kognic-io/pre_annotations.md:31]: Maybe we should stop talking about _dangling scenes_. The fact that a scene won't be annotated unless it has been added to a request doesn't make it _dangling_

File: docs-src/docs/kognic-io/project.md [hunk 3]
```diff
@@ -4,33 +4,28 @@ title: Projects
...
 
-In order to create inputs via the API, a Kognic project needs to exist.
-Projects are configured by the Kognic Professional Services team, during the Guideline Agreement Process (GAP) of a new client engagement.
+A Kognic [project](../key_concepts.md) must first be set in order to create inputs.
+Projects are usually configured by the Kognic Professional Services team, during the Guideline Agreement Process (GAP) of a new client engagement.
 
 ### List Projects
 
-Projects in your organization can be listed with the KognicIOClient
+All existing projects within in your organization can be listed by using the Python API KognicIOClient. Make sure the authentication are set (see [Authentication](../kognic_auth.md)).
```
Comment (olinoskar) [inline docs-src/docs/kognic-io/project.md:12]:
```suggestion
All existing projects within your organization can be listed by using the Python API KognicIOClient. Make sure the authentication are set (see [Authentication](../kognic_auth.md)).
```

File: docs-src/docs/kognic-io/project.md [hunk 4]
```diff
@@ -44,47 +39,66 @@ Ongoing projects can benefit from using batches in two ways
...
+client = KognicIOClient()
+project_batch = client.project.create_batch(
+  project="<project_external_id>",
+  batch="<batch_external_id>",
+)
 ```
-Creates a new batch in the `open` state. The new batch will contain the same Annotation Types
-(see [Annotation Types](annotation_types)) as 
-the latest previous batch, which means that the process of uploading inputs will be identical between batches.
+The newly created batch will contain the same Annotation Types (see [Annotation Types](annotation_types)) as
```
Comment (olinoskar) [inline docs-src/docs/kognic-io/project.md:68]:
```suggestion
The newly created batch will contain the same [Annotation Types](annotation_types) as 
```

File: docs-src/docs/kognic-io/pre_annotations.md (hunk 1 above)
Comment (Anqiqi-Li) [reply docs-src/docs/kognic-io/pre_annotations.md:27]: Done

File: docs-src/docs/kognic-io/pre_annotations.md (hunk 2 above)
Comment (Anqiqi-Li) [reply docs-src/docs/kognic-io/pre_annotations.md:31]: But I think the pre-annotation page has to be reviewed once more in another PR. Some more new questions popped up and I myself do not fully understand what is supported and what is not..

**Review by olinoskar — APPROVED** [review]

**Pull request merged**